napari/utils/events/dataclass.py

```diff
@@ -478,4 +527,6 @@ def evented_dataclass(
         # convert public dataclass fields to properties
         cls = convert_fields_to_properties(cls)
     setattr(cls, '_get_state', _dc.asdict)
```

Comment:
we can probably drop `_get_state` if we have `as_dict`

Reply:
done